src/App.css

```diff
@@ -2,30 +2,14 @@
   text-align: center;
 }
 
-.Logo-header {
-  text-align: center;
-  position: relative;
-  display: inline;
-}
-
-.App-logo-back {
-  background-color: #222;
-  position: relative;
-  height: 400px;
-  padding: 0px;
-}
-
 .App-logo {
-  animation: App-logo-spin infinite 30s linear;
-  position: absolute;
-  left: -40px;
-  top: -350px;
-  height: 280px;
+  animation: App-logo-spin infinite 20s linear;
```

Comment:
@beatfreaker Can these changes be retained. They are used by the Header elements in `App.js`